plugins/orchestrator/src/components/WorkflowDefinitionViewerPage/WorkflowDefinitionViewerPage.tsx

```diff
@@ -1,82 +1,100 @@
 import { Button, Grid } from '@material-ui/core';
 
+import { orchestratorApiRef } from '../../api';
 import {
+  editWorkflowRouteRef,
   executeWorkflowRouteRef,
   workflowDefinitionsRouteRef,
 } from '../../routes';
-import { BaseOrchestratorPage } from '../BaseOrchestratorPage/BaseOrchestratorPage';
-import { OrchestratorSupportButton } from '../OrchestratorSupportButton/OrchestratorSupportButton';
-import {
-  EditorViewKind,
-  WorkflowEditor,
-  WorkflowEditorRef,
-} from '../WorkflowEditor';
+import { EditorViewKind, WorkflowEditor } from '../WorkflowEditor';
+import WorkflowDefinitionDetailsCard from './WorkflowDefinitionDetailsCard';
 
 export const WorkflowDefinitionViewerPage = () => {
-  const [name, setName] = useState<string>();
   const { workflowId, format } = useRouteRefParams(workflowDefinitionsRouteRef);
-  const [workflowEditor, workflowEditorRef] =
-    useController<WorkflowEditorRef>();
-  const [loading, setLoading] = useState(true);
+
+  const orchestratorApi = useApi(orchestratorApiRef);
+  const { loading, value: workflowOverview } = useAsync(() =>
+    orchestratorApi.getWorkflowOverview(workflowId),
+  );
+
   const navigate = useNavigate();
   const executeWorkflowLink = useRouteRef(executeWorkflowRouteRef);
-
+  const editWorkflowLink = useRouteRef(editWorkflowRouteRef);
   const workflowFormat = useMemo(
     () => (format === 'json' ? 'json' : 'yaml'),
     [format],
   );
 
-  useEffect(() => {
-    if (!workflowEditor?.workflowItem) {
-      return;
-    }
-    setLoading(false);
-    setName(workflowEditor.workflowItem.definition.name);
-  }, [workflowEditor]);
-
-  const onExecute = useCallback(() => {
+  const onExecute = () => {
     navigate(executeWorkflowLink({ workflowId }));
-  }, [executeWorkflowLink, navigate, workflowId]);
+  };
+
+  const onEdit = () => {
+    navigate(editWorkflowLink({ workflowId, format }));
+  };
 
   return (
-    <BaseOrchestratorPage>
-      <ContentHeader title="Definition">
-        <OrchestratorSupportButton />
-      </ContentHeader>
-      <Grid container spacing={3} direction="column">
-        <Grid item>
-          {loading && <Progress />}
-          <InfoCard
-            title={name}
-            action={
-              <Button
-                color="primary"
-                variant="contained"
-                style={{ marginTop: 8, marginRight: 8 }}
-                onClick={() => onExecute()}
-              >
-                Execute
-              </Button>
-            }
-          >
-            <div style={{ height: '600px' }}>
-              <WorkflowEditor
-                ref={workflowEditorRef}
-                kind={EditorViewKind.EXTENDED_DIAGRAM_VIEWER}
-                workflowId={workflowId}
-                format={workflowFormat}
-              />
-            </div>
-          </InfoCard>
-        </Grid>
-      </Grid>
-    </BaseOrchestratorPage>
+    <ErrorBoundary>
```

Comment:
The package/app already wraps the plugins in an ErrorBoundary of its own.
Please try reusing the `BaseOrchestratorPage`.

Reply:
This doesn't include the typeLink attribute enabling breadcrumbs, and the viewer page does require the padding
I can add these as props of course but don't really see what advantage this reuse gives

Reply:
I would rather extend the `BaseOrchestratorPageProps` in order to support these use cases. The padding can be applied by removing the `noPadding` prop in the Content component wrapping the children prop (or just setting it to `false`). The breadcrumbs could be supported in a similar fashion I guess.

Reply:
Since the OrchestratorPage requires the noPadding, I made it a property of BaseOrchestratorPage 
as well